src/structs/types/cache_information.rs

```diff
@@ -147,6 +155,37 @@ impl Serialize for SMBiosCacheInformation<'_> {
     }
 }
 
+/// # Maximum memory capacity, in kilobytes, for this cache item
+#[derive(Serialize, Debug, PartialEq, Eq)]
+pub enum CacheMemorySize {
+    /// Maximum memory capacity in Kilobytes
+    Kilobytes(u64),
+    /// Use `maximum_cache_size_2` to retrieve the maximum capacity
+    SeeMaximumCacheSize2,
```

Comment:
`SeeMaximumCacheSize2` might also be `InstalledCacheSize2`.  This is not likely to confuse anyone based on the context of when it is used.  Or could go more vague with `SeeCacheSize2` which is no better or worse I think.  Using this library requires a degree of reading the SMBIOS standard.  A mitigation is to add to the comment on line 163 to include "or `installed_cache_size_2`"

Reply:
Good suggestions I'll clean this up a little.